aws-opentelemetry-distro/tests/amazon/opentelemetry/distro/test_otlp_sigv4_exporter.py

```diff
@@ -0,0 +1,301 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+import os
+from unittest import TestCase
+from unittest.mock import ANY, MagicMock, PropertyMock, patch
+
+import requests
+from botocore.credentials import Credentials
+
+from amazon.opentelemetry.distro.aws_opentelemetry_configurator import OTLPAwsSigV4Exporter
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import (
+    DEFAULT_COMPRESSION,
+    DEFAULT_ENDPOINT,
+    DEFAULT_TIMEOUT,
+    DEFAULT_TRACES_EXPORT_PATH,
+    OTLPSpanExporter,
+)
+from opentelemetry.exporter.otlp.proto.http.version import __version__
+from opentelemetry.sdk.environment_variables import OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
+from opentelemetry.sdk.trace import SpanContext, _Span
+from opentelemetry.trace import SpanKind, TraceFlags
+
+OTLP_CW_ENDPOINT = "https://xray.us-east-1.amazonaws.com/v1/traces"
```

Comment:
nit: Rename this accordingly as well (and where it's used), e.g.:
```suggestion
OTLP_XRAY_ENDPOINT = "https://xray.us-east-1.amazonaws.com/v1/traces"
```